rotkehlchen/db/filtering.py

```diff
@@ -1537,6 +1537,7 @@ def make(
 
 class AssetsFilterQuery(DBFilterQuery):
     ignored_assets_handling: IgnoredAssetsHandling = IgnoredAssetsHandling.NONE
+    hide_assets_threshold: FVal | None = None
```

Comment:
this filter is used in contexts where the price doesn't make sense. IT should be a different filter but the approach is not correct